x-pack/test/idp-fixture/src/main/resources/nginx/Dockerfile

```
@@ -0,0 +1,37 @@
+# We need to do SSL tunnelling, but NGINX doesn't support this out of the box - there's a module for this, but the only
+#  way to install it is to compile NGINX from source, so here we go
+
+FROM nginx:latest
```

Using latest works fine now, but it might break in the future. Let's fix it to the same version you are patching below.
```suggestion
FROM nginx:1.27.1
```